src/App.vue

```diff
@@ -19,6 +22,24 @@ import HelloI18n from './components/HelloI18n.vue';
   },
 })
 export default class App extends Vue {
+  client: AppGatewayServiceClient = undefined as any;
+
+  created() {
+    this.client = new AppGatewayServiceClient('http://localhost:50056', null, null);
+    this.getStatus();
+  }
+
+  getStatus() {
+    const request: AppGatewayServiceRequest = new AppGatewayServiceRequest();
+
+    // TODO need to make some sort of metadata interface as a type
+    const metadata = { 'custom-header-1': 'value1' };
```

Comment:
To authenticate you have to send a metadata to `app-gateway-svc`.
Check the following:
- review the epic especially step 3 in https://hwsc-org.github.io/wikis/app-gateway-svc/epics.html#authenticateuser
- https://github.com/hwsc-org/hwsc-api-blocks/blob/master/protobuf/hwsc-app-gateway-svc/hwsc-app-gateway-svc.js#L24
- note the base 64 encoding
- note the email & password from the 
config https://hwsc.slack.com/archives/CGF26TN1Z/p1555385177000700
- the email & password only allows `CreateUser`